simd/arm/aarch32/jchuff-neon.c

```diff
@@ -31,7 +31,7 @@
 #include "../../../jsimddct.h"
 #include "../../jsimd.h"
 #include "../jchuff.h"
-#include "neon-compat.h"
+#include "../neon-compat.h"
```

Comment:
A better fix would be to instead just add the arm include path to the lua file so we are not modifying upstream source.